### PR TITLE
amam/change_oauth

### PR DESCRIPTION
**Pull request by mustofa-binary**
* add `brand=deriv` to WS connection and oauth URL

### CODE REVIEW

File: src/common/login.js [hunk 1]
```diff
@@ -22,11 +22,11 @@ const Login = (() => {
             date_first_contact
                 ? `&date_first_contact=${date_first_contact}`
                 : ''
-        }`
+            }`
```
Comment (negarn) [inline src/common/login.js]: why add extra space?

File: src/common/login.js [hunk 2]
```diff
@@ -22,11 +22,11 @@ const Login = (() => {
...
 
         return server_url && /qa/.test(server_url)
-            ? `https://${server_url}/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}`
-            : `https://oauth.deriv.com/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}`
+            ? `https://${server_url}/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=deriv`
+            : `https://oauth.deriv.com/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=deriv`
```
Comment (negarn) [inline src/common/login.js]: prefer storing it in a variable similar to previous queries

File: src/common/login.js (hunk 1 above)
Comment (mustofa-binary) [reply src/common/login.js]: we have autosave prettier

File: src/common/utility.js [hunk 3]
```diff
@@ -47,10 +47,12 @@ class PromiseClass {
     }
 }
 
-const DERIV_APP_URL = 'https://deriv.app'
+const deriv_app_url = 'https://deriv.app'
+const deriv_name = 'Deriv'
```
Comment (ashkanx) [inline src/common/utility.js]:
```suggestion
const brand_name = 'Deriv'
```

File: src/common/login.js [hunk 4]
```diff
@@ -22,11 +23,11 @@ const Login = (() => {
             date_first_contact
                 ? `&date_first_contact=${date_first_contact}`
                 : ''
-        }`
+            }`
 
         return server_url && /qa/.test(server_url)
-            ? `https://${server_url}/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}`
-            : `https://oauth.deriv.com/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}`
+            ? `https://${server_url}/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${deriv_name.toLowerCase()}`
```
Comment (ashkanx) [inline src/common/login.js]:
```suggestion
            ? `https://${server_url}/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${brand_name.toLowerCase()}`
```

File: src/common/login.js [hunk 5]
```diff
@@ -22,11 +23,11 @@ const Login = (() => {
...
+            : `https://oauth.deriv.com/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${deriv_name.toLowerCase()}`
```
Comment (ashkanx) [inline src/common/login.js]:
```suggestion
            : `https://oauth.deriv.com/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${brand_name.toLowerCase()}`
```

File: src/common/utility.js [hunk 6]
```diff
@@ -47,10 +47,12 @@ class PromiseClass {
...
 
 export {
-    DERIV_APP_URL,
+    deriv_app_url,
+    deriv_name,
```
Comment (ashkanx) [inline src/common/utility.js]:
```suggestion
    brand_name,
```

File: src/common/websocket/socket_base.js [hunk 7]
```diff
@@ -3,6 +3,7 @@
 import { State } from '../storage'
 import {
     cloneObject,
+    deriv_name,
```
Comment (ashkanx) [inline src/common/websocket/socket_base.js]:
```suggestion
    brand_name,
```

File: src/common/websocket/socket_base.js [hunk 8]
```diff
@@ -27,7 +28,7 @@ const BinarySocketBase = (() => {
 
     // TODO: add language
     // const socket_url = `${getSocketURL()}?app_id=${getAppId()}&l=${getLanguage()}`;
-    const socket_url = `${getSocketURL()}?app_id=${getAppId()}&l=en`
+    const socket_url = `${getSocketURL()}?app_id=${getAppId()}&l=en&brand=${deriv_name.toLowerCase()}`
```
Comment (ashkanx) [inline src/common/websocket/socket_base.js]:
```suggestion
    const socket_url = `${getSocketURL()}?app_id=${getAppId()}&l=en&brand=${brand_name.toLowerCase()}`
```

File: src/common/login.js [hunk 9]
```diff
@@ -2,6 +2,7 @@ import Cookies from 'js-cookie'
 import { isStorageSupported, LocalStore } from './storage'
 import TrafficSource from './traffic-source'
 import isMobile from './os-detect'
+import { deriv_name } from './utility'
```
Comment (ashkanx) [inline src/common/login.js]:
```suggestion
import { brand_name } from './utility'
```

**Pull request merged**